contracts/interchain-token-service/src/contract/migrations/mod.rs

```diff
@@ -1,16 +1,20 @@
 use axelar_wasm_std::error::ContractError;
 use axelar_wasm_std::migrate_from_version;
+use cosmwasm_schema::cw_serde;
 #[cfg(not(feature = "library"))]
 use cosmwasm_std::entry_point;
-use cosmwasm_std::{DepsMut, Empty, Env, Response};
+use cosmwasm_std::{DepsMut, Env, Response};
 
 pub mod v1_1_0;
 
-pub type MigrateMsg = Empty;
+#[cw_serde]
+pub struct MigrateMsg {
+    operator_address: String,
```

Comment:
This should be set to the same address as the prover admin fyi for creating the release doc